### ISSUE TITLE
Make healthcheck optional

### ISSUE THREAD

**Issue by sUyMur** [bubuntux/nordvpn#84]
**Is your feature request related to a problem? Please describe.**
After NordVPN loses its connection I have to restart the stack behind it. 

**Describe the solution you'd like**
Including the healthcheck in docker run / docker-compose would make it editable. 
My main concern here is the default actoin when unprotected: `nordvpn connect` - Therefore the container will never be unhealthy and therefore I can't restart my stack.

**Describe alternatives you've considered**
I tried overwriting the healthcheck in docker-compose, but that didn't work. 
I can curl / ping website from other containers behind NordVPN but somehow other services are impacted.
Also, making the healthcheck variable would an alternative: e.g. removing `nordvpn connect`, when there is no connection would mean it would get the status unhealthy, right?

**Additional context**
Spinning up all services with one docker-compose doesn't work as expected: since e.g. qBittorrent won't connect. 
There has to delay between starting NordVPN and dependent services.
My solution originally was to monitor NordVPNs health and if unhealthy restart NordVPN, wait 30 seconds, restart my stack. And the initial spin up would involve at least two docker-compose files, so i can insert a delay between starting NordVPN and subsequent services.

This could be achieved, if the healthcheck would be "optional" or rather in the docker-compose file.
This would also bump the docker-compose version to 3.4 instead of 3, so `start_period` would be usable.

**Comment by sUyMur** [comment]
I have forked it and made my suggested edits:
Dockerfile: removed / commented `HEALTHCHECK`

Added:
`NordVPN-compose.yaml`
`Stack-compose.yaml` (includes: qBittorrent, radarr, sonarr, jackett as an example setup)

`HOST_checkServerHealth.sh`: 
- script to check for health status of containers (when unhealthy, `exec HOST_startDocker.sh`)

`HOST_startDocker.sh`: 
- stops all docker containers 
- restarts `NordVPN-compose.yaml` and `Stack-compose.yaml`. (Where `Stack-compose.yaml` waits for NordVPN to be `healthy`)

**Comment by bubuntux** [comment]
docker allow you to override the healthcheck 

https://docs.docker.com/compose/compose-file/#healthcheck
https://docs.docker.com/engine/reference/builder/#healthcheck

the container will become unhealthy when is not connected and is not able to re-connect.

>  I tried overwriting the healthcheck in docker-compose, but that didn't work.
how did you test it? can you provide example ?

**Comment by sUyMur** [comment]
Sorry for not responding sooner. 
I forgot about it, as I just found a solution that fit my needs.

My main problem was that my stack behind NordVPN depends on NordVPN having a connection.
Once that connection dropped, I have to restart the stack behind the NordVPN container.Therefore, I just have this as a startup routine:
docker-compose -f /path/to/NordVPN/docker-compose.yaml up -d
sleep 120
docker-compose -f /path/to/stackBehindNordVPN/docker-compose.yaml up -d

To monitor health, I wrote a small script that checks for NordVPN being unhealthy - basically:
```
while [ 1=1 ]
do
sleep 180
docker ps | grep "NordVPN" | grep "unhealthy" && ./startDockerStack.sh
enddo
```

This was my bad for not understanding the docker documentation. 
Sorry, and thanks for letting me know. :-)
